services/spar/src/Spar/API.hs

```diff
@@ -264,11 +266,18 @@ validateNewIdP _idpMetadata _idpeTeam = do
 
 -- | Type families to convert spar's 'API' type into an "outside-world-view" API type
 -- to expose as swagger docs intended to be used by client developers.
+--
 -- Here we assume the 'spar' service is only accessible from behind the 'nginz' proxy, which
 --   * does not expose routes prefixed with /i/
 --   * handles authorization (adding a Z-User header if requests are authorized)
 --   * does not show the swagger end-point itself
-type OutsideWorldAPI = StripSwagger (StripInternal (StripAuth API))
+--
+-- In addition, we strip the SCIM documentation because SCIM doesn't have
+-- Swagger schemas yet.
+--
+-- TODO: consider exposing the SCIM docs as well.
```

Comment:
Yes, i think just having the generic stuff would already by quite helpful.

Reply:
In this case do you think Swagger schemas should live in hscim, or be defined in Spar? (My vote is for the former but maybe there are some drawbacks)

Reply:
i agree, they should live in hscim.  if there are any problems with that neither of us can think of, we can refactor later.